src/lvl_script.c

```diff
@@ -158,6 +159,8 @@ const struct CommandDesc command_desc[] = {
   {"CHANGE_SLAB_TYPE",                  "NNS     ", Cmd_CHANGE_SLAB_TYPE},
   {"IF_SLAB_OWNER",                     "NNP     ", Cmd_IF_SLAB_OWNER},
   {"IF_SLAB_TYPE",                      "NNS     ", Cmd_IF_SLAB_TYPE},
+  {"NEUTRAL_ENTRANCE_LEVEL",            "N       ", Cmd_NEUTRAL_ENTRANCE_LEVEL},
```

Comment:
These two I won't accept like this of course.  Stuff from prototypes that sucked so much it did not make it into the final game, will not be put into keeperfx either.

I'm not even sure what creature_inby should do for you, but an entrance level for all units would be a script command I accept, provided you give it a better name.

Reply:
creature_inby lets you (re)set the creature pool for the set creature, rather than just add to it. It occurred to me that it's highly unlikely to be an exact equivalent of ADD_CREATURE_TO_POOL, because the name doesn't really fit.

Reply:
Does ADD_CREATURE_TO_POOL allow negative values? But even if now, I see absolutely no reason why you chose CREATURE_INBY over 'SET_CREATURE_POOL' or something.

Reply:
I assume so, though I haven't actually tested it. CREATURE_INBY seems to be what Bullfrog called it, so I just went with that.

Reply:
No, bullfrog went with ADD_CREATURE_TO_POOL. And both they, you and I know CREATURE_INBY is a dumb name that describes nothing.

Reply:
It actually kind of makes sense if you know what 'inby' means. But there doesn't seem to be a way to reset the pool once it's set (you can only add to it and possibly subtract from it), and I can only assume that's what this command did.

Reply:
If you subtract the pool to 0 and add it to the new number, you've reset it. I don't really see any cases where you would prefer to reset over adding/subtracting to the pool though, do you?
And no, 'if you know what it does it kinda makes sense' is nowhere near good enough as a name.

Reply:
But how can you get the value of the pool from the script? 

We may not be able to envision such situations, but that doesn't mean others won't. The level maker should be able to do what they want, shouldn't they?

Reply:
Adding options is good, but in the end we're trying to think of the most potent options. Adding new script commands with huge overlap with other script commands which might not even add a single new use case at all is not very smart, especially not when comparing it with adding script commands that enable new scenarios. Making a huge list of Script commands makes it more time consuming for map makers to learn the script commands, so each one should have value.

In most maps you just set the pool, so you know what the pool is. You could do a map, where based on conditions you add to the pool and even though you could script it out to always know what the pool is, that would require quite a bit of scripting. In most cases this is linear and you would still know, but in others you can just subtract a large number from the pool to always set it to 0, then add the number you want to set it to.

Reply:
But subtracting more than the value of the pool doesn't result in 0; you'll get a negative value. Therefore, you cannot simply add the number you want to "reset" the pool to.